web/src/locales/cs.json

```diff
@@ -94,41 +94,41 @@
     "mapColorBlind-question": "Jsem barvoslepý. Jak můžu číst vaši mapu?",
     "mapColorBlind-answer": "Můžete změnit barvy na mapě tím, že zapnete režim pro barvoslepé. Toto nastavení se nachází ve spodní části levého panelu v módu prohlížeče a na záložce INFO v mobilní aplikaci."
   },
   "mapAreas": {
     "groupName": "Oblasti na mapě",
     "noData-question": "Proč nejsou data z mé oblasti dostupná?",
-    "noData-answer": "Buď je zdroj dat dočasně nedostupný, nebo protože zatím nemáme žádné zdroje dat pro tuto oblast. Můžete <a href=\"#contribute\" class=\"entry-link\">nám pomoci přidáním nových zdrojů dat</a>.",
+    "noData-answer": "Buď je zdroj dat dočasně nedostupný, nebo protože zatím nemáme žádné zdroje dat pro tuto oblast. Můžete <a href='#contribute' class='entry-link'>nám pomoci přidáním nových zdrojů dat</a>.",
     "whySmallAreas-question": "Proč dělíte svět do malých oblastí a raději nezobrazíte průměry států?",
     "whySmallAreas-answer": "Tyto oblasti jsou nejmenší geografické oblasti, pro která máme data. Díky přístupu k informacím na nejjemnější úrovni, může mít spotřebitel energie v různých oblastech přesnější přehled o původu elektřiny, kterou spotřebovává a s tím spojeným dopadem na klima.",
     "divideExistingArea-question": "Můžete rozdělit mou oblast na menší?",
-    "divideExistingArea-answer": "Určitě, když existují separátní zdroje dat pro každou část dané oblasti. Můžete <a href=\"#contribute\" class=\"entry-link\">nám pomoci přidáním nových datových zdrojů</a>.",
+    "divideExistingArea-answer": "Určitě, když existují separátní zdroje dat pro každou část dané oblasti. Můžete <a href='#contribute' class='entry-link'>nám pomoci přidáním nových datových zdrojů</a>.",
     "seeHistoricalData-question": "Můžu zobrazit historii oblasti starší než 24 hodin?",
-    "seeHistoricalData-answer": "Můžete si zakoupíte přístup do <a href=\"https://electricitymaps.com?utm_source=app.electricitymaps.com&utm_medium=referral\" target=\"_blank\">databáze historických dat</a>."
+    "seeHistoricalData-answer": "Můžete si zakoupíte přístup do <a href='https://electricitymaps.com?utm_source=app.electricitymaps.com&utm_medium=referral' target='_blank'>databáze historických dat</a>."
   },
   "methodology": {
     "groupName": "Metogologie",
-    "carbonIntensity-question": "Co je \"uhlíková intenzita\"?",
+    "carbonIntensity-question": "Co je 'uhlíková intenzita'?",
```

Comment:
IDK about the grammar in various languages but do you think this could be a problem?

It will be significantly harder to fix if it is but I also don't want to make something "wrong" in languages just because we didn't know about it.

Reply:
I don't know if it will be a problem. @tonypls or @Alportan do have any knowledge about this?

Reply:
No idea! I think it should be okay to proceed and see if we get any feedback on the languages we're not sure about